### DISCUSSION TITLE
Display of the last triggered time in full format

### DISCUSSION THREAD

**Discussion by Tsarre** [home-assistant/frontend#18968] [Automations / scripts / scenes] [answered]
Sometimes something happens in the house that you do not expect (some automation / script that is executed unintentionally). In the past, I could simply note the time and later use Home Assistant to investigate what was being done at that time.
This is currently not possible, as it is not the time but the time that has passed that is displayed (which is even rounded to hours after one hour).
So please use the format days/hours/minutes/seconds again.

Same thing on a smartphone, but also with the possibility to order it on time (now only sorting on name is possible).

![Schermafbeelding 2023-12-08 131156](https://github.com/home-assistant/frontend/assets/95714944/a3c34f83-b1ba-4f80-9376-0b81bea50a91)

**Comment by karwosts** [comment]
I guess relative vs absolute time is somewhat subjective, so I won't personally suggest to change that, but I will point out if you do need to see the absolute time, if you click the automation and then hit the `Traces` button, the first trace will show the exact execution time (and the reason it was executed)

**Comment by Tsarre** [comment]
I personally don't see any advantage of displaying a relative time, but maybe both can be displayed.

**Comment by yank555-lu** [comment]
I agree with Tsarre, relative time is very unprecise after one hour already.

And the possibility to sort the last executed time can come in handy, too, tbh.

**Reply by karwosts** [reply]
> And the possibility to sort the last executed time can come in handy, too, tbh.

Last triggered _is_ last executed.

**Reply by yank555-lu** [reply]
I know it's the same, the focus was on the word 'SORT' as Tsarre wrote, I agree with him on that it could be usefull to SORT on when last TRIGGERED if you prwfer the wording.

**Reply by karwosts** [reply]
You can already sort on the last triggered time, so I'm not sure why that's being asked for...

**Reply by yank555-lu** [reply]
You're right, my bad, I was here because of the  unprecise relative time ...

My last sentence was simply because of what Tsarre wrote, I should have checked instead of assuming that was not sortable, sorry for that and for wasting your time on this.

The unprecise relative time point still stands, though.

**Comment by ildar170975** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/28158

<img width="778" height="186" alt="519453524-c87740df-8065-41a9-ac7d-5d392bba66b1" src="https://github.com/user-attachments/assets/ac594715-6e58-4a34-b397-32bd71605216" />

Consider to mark as "solved" if this does solve the main problem.

**Comment by marcinbauer-ohf** [comment]
We do use dynamic labeling for the Last triggered time. If it's above a certain amount of time we display the absolute date and time:
<img width="1049" height="1291" alt="CleanShot 2025-12-02 at 12 51 16" src="https://github.com/user-attachments/assets/c1772b4a-e8ac-4219-8a29-6a2e64490f33" />
I think @ildar170975 solution works well here. Thanks!

**Comment by ildar170975** [comment]
Can be marked as solved in 2026.1
